test/py/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ function(add_py_venv_fixture FIXTURE_NAME VIRTUAL_ENV_DIR REQUIREMENTS_FILE)
         set(PYTHON_EXECUTABLE ${PYTHON_${PYTHON_VERSION}_EXECUTABLE})
 
         if(NOT TEST py_${PYTHON_VERSION}_${FIXTURE_NAME}_initialize_env)
-            if (NOT (${FIXTURE_NAME} STREQUAL "onnx" AND ${PYTHON_VERSION} STREQUAL ${PYTHON_VERSION_TO_DISABLE_ONNX}))
+            if (NOT (${PYTHON_VERSION} STREQUAL ${PYTHON_VERSION_TO_DISABLE_ONNX}))
```

Comment:
We should only disable the test when `MIGRAPHX_DISABLE_VIRTUAL_ENV=Off`.

Reply:
Thanks for the feedback Paul. I will make the changes as you suggested in a follow up PR.

Reply:
I looked at initially disabling the tests using `MIGRAPHX_DISABLE_VIRTUAL_ENV=On` for Python 3.6 but I had some concerns regarding the code flow if we disable it for a particular Python version inside CMakeLists.txt. I can discuss this on Monday and I can proceed based on the feedback.